shared/searchv3/dumb.js

```diff
@@ -338,19 +341,60 @@ const userInputMap: DumbComponentMap<UserInput> = {
         style: {
           width: 460,
           padding: 4,
-          boxShadow: '0 0 3px rgba(0, 0, 0, .25)',
+          borderWidth: 2,
+          borderColor: 'gray',
+          borderStyle: 'solid',
         },
       },
       userItems: chrisUsers,
       usernameText: 'Chris Hemswor',
-      showAddButton: false,
     },
   },
 }
 
+const UserInputEditable = compose(
+  withState('usernameText', 'onChangeText', ''),
+  withState('userItems', 'setUserItems', ({userItems}) => userItems),
```

Comment:
This uses the prop (provided by dumb sheet) to set the initial value of the state. It's a bit terse, but I think it's the way you do it in recompose land. I could rename it to `defaultUserItems`, if you'd prefer?